packages/story-editor/src/components/videoTrim/provider.js

```diff
@@ -40,6 +49,25 @@ function VideoTrimProvider({ children }) {
     resetOffsets,
   } = useVideoNode();
 
+  const performTrim = useCallback(() => {
+    const { resource } = selectedElements[0];
+    if (!resource) {
+      return;
+    }
+    trimExistingVideo({
+      resource,
```

Comment:
```suggestion
    const length = (endOffset * 1000) - (startOffset * 1000);
    trimExistingVideo({
      resource: {
          ...resource, 
          length: (endOffset * 1000) - (startOffset * 1000)
          lengthFormatted: formatMsToHMS(endOffset - startOffset)
      },
```

We could do this, so preview time stamp works workly.

Reply:
Ah, good point, thanks!

Reply:
I worked on a PR - https://github.com/google/web-stories-wp/pull/9143